extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1521,6 +1521,17 @@ namespace Microsoft.Windows.DevHome.SDK
         };
     }
 
+    // Used by the extension to provide information on the chat style generation
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 6)]
+    runtimeclass QuickStartChatStyleResult
+    {
+        QuickStartChatStyleResult(String chatResponse);
+
+        // Allows extensions to provide intermediary status for the operation e.g "Generating project".
```

Comment:
Is this for status or for the chat message?

Reply:
Should be for chat message. Should be fixed now.